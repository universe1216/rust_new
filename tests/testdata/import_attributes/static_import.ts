import data1 from "./data.json" with { type: "json" };
// deno-lint-ignore no-import-assertions
import data2 from "./data.json" assert { type: "json" };

console.log(data1);
console.log(data2);
