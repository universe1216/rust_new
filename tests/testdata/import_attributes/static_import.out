⚠️  Import assertions are deprecated. Use `with` keyword, instead of 'assert' keyword.

import data2 from "./data.json" assert { type: "json" };

  at file:///[WILDLINE]

{ a: "b", c: { d: 10 } }
{ a: "b", c: { d: 10 } }
